purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/CustomerCenterConfigData.kt

```diff
@@ -0,0 +1,465 @@
+                    if (title != other.title) return false
+                    if (subtitle != other.subtitle) return false
+
+                    return true
+                }
+
+                override fun hashCode(): Int {
+                    var result = androidOfferId.hashCode()
+                    result = HASH_CODE_MULTIPLIER * result + eligible.hashCode()
+                    result = HASH_CODE_MULTIPLIER * result + title.hashCode()
+                    result = HASH_CODE_MULTIPLIER * result + subtitle.hashCode()
+                    return result
+                }
+            }
+
+            @Serializable
+            class FeedbackSurvey(
+                val title: String,
+                val options: List<Option>,
+            ) : PathDetail() {
+                @Serializable
+                class Option(
+                    val id: String,
+                    val title: String,
+                    @SerialName("promotional_offer") val promotionalOffer: PromotionalOffer? = null,
+                ) {
+                    override fun equals(other: Any?): Boolean {
+                        if (this === other) return true
+                        if (javaClass != other?.javaClass) return false
+
+                        other as Option
+
+                        if (id != other.id) return false
+                        if (title != other.title) return false
+                        if (promotionalOffer != other.promotionalOffer) return false
+
+                        return true
+                    }
+
+                    override fun hashCode(): Int {
+                        var result = id.hashCode()
+                        result = 31 * result + title.hashCode()
+                        result = 31 * result + (promotionalOffer?.hashCode() ?: 0)
+                        return result
+                    }
+                }
+
+                override fun equals(other: Any?): Boolean {
+                    if (this === other) return true
+                    if (javaClass != other?.javaClass) return false
+
+                    other as FeedbackSurvey
+
+                    if (title != other.title) return false
+                    if (options != other.options) return false
+
+                    return true
+                }
+
+                override fun hashCode(): Int {
+                    var result = title.hashCode()
+                    result = HASH_CODE_MULTIPLIER * result + options.hashCode()
+                    return result
+                }
+            }
+        }
+
+        @Serializable
+        enum class PathType {
+            @SerialName("MISSING_PURCHASE")
+            MISSING_PURCHASE,
+
+            @SerialName("REFUND_REQUEST")
+            REFUND_REQUEST,
+
+            @SerialName("CHANGE_PLANS")
+            CHANGE_PLANS,
+
+            @SerialName("CANCEL")
+            CANCEL,
+
+            @SerialName("UNKNOWN")
+            UNKNOWN,
```

Comment:
Are these uppercase in the JSON? If so, these `@SerialName`s are redundant right?